src/Microsoft.Data.SqlClient/tests/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj

```diff
@@ -43,11 +42,10 @@
     <Compile Include="SqlBulkCopyColumnMappingCollectionTest.cs" />
     <Compile Include="AmbientTransactionFailureTest.cs" />
     <Compile Include="SqlConnectionTest.RetrieveStatistics.cs" />
-    <Compile Include="FakeDiagnosticListenerObserver.cs" />
     <Compile Include="SqlMetaDataTest.cs" />
     <Compile Include="SqlConnectionStringBuilderTest.cs" />
     <Compile Include="SerializeSqlTypesTest.cs" />
-    <!-- <Compile Include="DiagnosticTest.cs" /> -->
+    <Compile Include="TestTdsServer.cs" />
```

Comment:
Isn't this one already in the manual tests?

Reply:
This is functional test. Diagnostic test was in that folder. We moved to Manual tests as it is using connection string from json file.

Reply:
OK. If `TestTdsServer` belongs to here, why we have it in `Microsoft.Data.SqlClient.ManualTesting.Tests.csproj` ? Duplicate?

Reply:
yes it is a duplicate.

Reply:
See my comments in `Microsoft.Data.SqlClient.ManualTesting.Tests.csproj`